docs/pages/getting_started/usage.md

```diff
@@ -209,33 +283,53 @@ auto g = sparsebase::object::Graph<vertex_type, edge_type, value_type>(data);
 Alternatively we can create a graph by directly passing the reader.
 
 ```cpp
- sparsebase::object::Graph<vertex_type, edge_type, value_type> g;
- g.read_connectivity_to_coo(sparsebase::MTXReader<vertex_type, edge_type, value_type>(file_name));
+sparsebase::object::Graph<vertex_type, edge_type, value_type> g;
+g.read_connectivity_to_coo(sparsebase::MTXReader<vertex_type, edge_type, value_type>(file_name));
 ```
 
-AsAbsolute of the current version of the library, graphs function as containers of sparse data. However, there are plans to expand this in future releases.
+As of the current version of the library, graphs function as containers of sparse data. However, there are plans to expand this in future releases.
 
 ## Ordering
 
-Various orderings can be generated for a graph using the ``ReorderPreprocessType`` classes. 
-Currently these include ``RCMReoder`` and ``DegreeReorder``. There is also a ``GenericReorder`` class
-allowing the users to define their own custom orderings. For more details, please see the examples
-in the Github repository.
+Sparse data formats can be reordered easily using the `ReorderBase` class.
+
+```cpp
+sparsebase::context::CPUContext cpu_context;
+IDType* order = ReorderBase::Reorder<DegreeReorder>(format, {&cpu_context}, true);
```

Comment:
Actually, `DegreeReorder` requires a `DegreeReorderParams` containing an `ascending` boolean.

Reply:
Hmmm. I thought they were optional. I see now that they are not. I will change it but not sure if I like it.